soca/obs/icec_amsr2_north.yaml

```diff
@@ -0,0 +1,37 @@
+- obs space:
+    name: icec_amsr2_north
+    distribution:
+      name: *obs_distribution
+    obsdatain:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/icec_amsr2_north.{{window_begin}}.nc4
+    obsdataout:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/$(experiment).icec_amsr2_north.{{window_begin}}.nc4
+    simulated variables: [sea_ice_area_fraction]
+  obs operator:
+    name: Identity
+  obs error:
+    covariance model: diagonal
+  _letkf: &letkf
+    # note, this is only used for LETKF. If running with LETKF, the workflow
+    # will append "<< : *letkf" to the end of this file
+    obs localizations:
+    - localization method: Horizontal Gaspari-Cohn
+      lengthscale: 200e3
+  obs filters:
+  - *obs_land_mask
+  # Passivate obs where ocean fraction is > 90%
+  - filter: Domain Check
+    action:
+      name: passivate
+    where:
+    - variable: {name: sea_area_fraction@GeoVaLs}
+      maxvalue: 0.9
+  - filter: Domain Check
+    where:
+    - variable: { name: sea_surface_temperature@GeoVaLs}
```

Comment:
distance_from_coast@GeoVals filter is not added? Do you think it is needed for icec?